tasks/broad/Utilities.wdl

```diff
@@ -219,6 +219,10 @@ task ErrorWithMessage {
   runtime {
     docker: "ubuntu:20.04"
   }
+
+  output {
+    String throw_away_output = "throw_away"
```

Comment:
hopefully this doesn't break anything else in warp. it shouldn't, right? you're not under any obligation to use a task output.

Reply:
yea it shouldnt break anything, i can change to a boolean though